system-container/root/exports/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "version": "1.0",
+    "defaultValues": {
+        "OPTS": "",
+        "VAR_LIB_OPENSHIFT_INSTALLER" : "/var/lib/openshift-installer",
+        "VAR_LOG_OPENSHIFT_LOG": "/var/log/ansible.log",
+        "PLAYBOOK_FILE": "/usr/share/ansible/openshift-ansible/playbooks/byo/config.yml",
```

Comment:
Now that I understand this better I see more pros than cons, so ok for me to bring it back - I'd only ask that the comment around the symlink explains the where the requirement comes from a bit more (i.e. mention that the default `PLAYBOOK_FILE` must be specified with full path due to `cwd` in `config.json.template`...)